website/client/components/inventory/stable/petItem.vue

```diff
@@ -78,6 +111,43 @@ div
       click () {
         this.$emit('click', {});
       },
+      getPetItemClass (pet) {
```

Comment:
why is this a method? since the passed argument (`pet`) can be substitued with a `this.item` it could be a computed property